constants.go

```diff
@@ -262,6 +262,9 @@ const (
 	// ComponentWindowsDesktop is a Windows desktop access server.
 	ComponentWindowsDesktop = "windows_desktop"
 
+	// ComponentDynamicWindowsDesktop is a Windows desktop access server.
```

Comment:
This comment is exactly the same as the one on `ComponentWindowsDesktop`. I would expect the godoc to help me understand what is different about a dynamic desktop.

Then again, if this is just used for logging, then you can probably use the original component field.